### PR TITLE
HP Pavilion dv6.xml

### PR DESCRIPTION
**Pull request by ranyyy**
I fixed major & minor issues with the config you provided users of the Pavilion dv6.
First the critical temp is now 90C, this is because this laptop shuts down when reaching >100C.
Second I changed the temp. threshold.
Lastly I changed the fan speed so the laptop fan would be able to run at its max (my laptop had lots of thermal shutdowns when using your config, temp used to be 100C. Now it is 80C. Plus if I keep at 100% it'll be 60C).

### CODE REVIEW

File: Configs/HP Pavilion dv6.xml [hunk 1]
```diff
@@ -3,18 +3,53 @@
...
-  <CriticalTemperature>75</CriticalTemperature>
+  <CriticalTemperature>90</CriticalTemperature>
   <FanConfigurations>
     <FanConfiguration>
       <ReadRegister>177</ReadRegister>
       <WriteRegister>177</WriteRegister>
       <MinSpeedValue>9</MinSpeedValue>
-      <MaxSpeedValue>17</MaxSpeedValue>
+      <MaxSpeedValue>23</MaxSpeedValue>
+      <IndependentReadMinMaxValues>false</IndependentReadMinMaxValues>
```
Comment (ranyyy) [inline Configs/HP Pavilion dv6.xml]: It's disabled :)

File: Configs/HP Pavilion dv6.xml [hunk 2]
```diff
@@ -3,18 +3,53 @@
...
+      <MinSpeedValueRead>9</MinSpeedValueRead>
```
Comment (ranyyy) [inline Configs/HP Pavilion dv6.xml]: IGNORE IT

File: Configs/HP Pavilion dv6.xml [hunk 3]
```diff
@@ -3,18 +3,53 @@
...
+      <MaxSpeedValueRead>20</MaxSpeedValueRead>
```
Comment (ranyyy) [inline Configs/HP Pavilion dv6.xml]: THAT ONE TOO!

File: Configs/HP Pavilion dv6.xml [hunk 4]
```diff
@@ -3,18 +3,53 @@
...
       <ResetRequired>true</ResetRequired>
       <FanSpeedResetValue>0</FanSpeedResetValue>
-      <TemperatureThresholds />
+      <FanDisplayName />
```
Comment (ranyyy) [inline Configs/HP Pavilion dv6.xml]: Thresholds :)

**Comment by hirschmann** [comment]
Thanks for your improvements :+1:

**Pull request merged**